pkg/reconciler/revision/config/store_test.go

```diff
@@ -79,20 +77,23 @@ func TestStoreLoadWithContext(t *testing.T) {
 	})
 
 	t.Run("observability", func(t *testing.T) {
-		expected, _ := metrics.NewObservabilityConfigFromConfigMap(observabilityConfig)
+		expected, _ := o11yconfigmap.Parse(observabilityConfig)
 		if diff := cmp.Diff(expected, config.Observability); diff != "" {
 			t.Error("Unexpected observability config (-want, +got):", diff)
 		}
 
 		// Default config.
-		want, _ := metrics.NewObservabilityConfigFromConfigMap(&corev1.ConfigMap{Data: map[string]string{}})
-		got, err := metrics.NewObservabilityConfigFromConfigMap(observabilityConfigExample)
+		want, _ := o11yconfigmap.Parse(&corev1.ConfigMap{Data: map[string]string{}})
+		got, err := o11yconfigmap.Parse(observabilityConfigExample)
 		if err != nil {
 			t.Fatal("Error parsing example observability config:", err)
 		}
 
 		// Compare with the example and allow the log url template to differ
```

Comment:
nit since we are ignoring more fields
```suggestion
		// Compare with the example and allow the log url template, base config, and request metrics to differ
```